OpenRA.Mods.Common/Traits/Buildable.cs

```diff
@@ -35,10 +36,13 @@ public class BuildableInfo : TraitInfo<Buildable>
 		[Desc("Force a specific faction variant, overriding the faction of the producing actor.")]
 		public readonly string ForceFaction = null;
 
-		[SequenceReference]
+		[SequenceReference(factionSuffix: true)]
 		[Desc("Sequence of the actor that contains the icon.")]
 		public readonly string Icon = "icon";
 
+		[Desc("Should faction suffix be added for sequence " + nameof(Icon) + "?")]
+		public readonly bool UseFactionSuffix = false;
```

Comment:
```suggestion
		public readonly bool AddIconFactionSuffix = false;
```

Perhaps something like this makes more sense? Looking at yaml the variable is unclear otherwise